src/vsr/replica.zig

```diff
@@ -7376,52 +7406,16 @@ pub fn ReplicaType(
                 }
             }
 
-            // Request enough prepares to utilize our max IO depth:
-            var budget = self.journal.writes.available();
-            if (budget == 0) {
-                log.debug("{}: repair_prepares: waiting for IOP", .{self.replica});
-                return;
-            }
-
-            // Iterate through op_min..=self.op, but make sure to iterate commit_min+1..=self.op
-            // sub range first:
             // - our first priority is to commit further,
             // - afterwards, repair committed prepares which are at risk of getting evicted from
             //   the journal, to help repair any lagging replicas.
```

Comment:
Maybe give this list a new introduction comment? or reformat it so that it makes sense